jeos installation (cleanup) (#623)
SUSE/doc-susemanager: modules/installation/pages/install-vm.adoc

| Version:            | SLES_version_-JeOS-for-kvm-and-xen.x86_64-GM.qcow2
| Memory:             | 4096 MB
| CPU's:              | 2
| Storage Format:     | _.qcow2_ 24 GB (Default) JeOS Root Partition
| Virtual Disks:      |
| VirtIO Disk 2       | 101 GB for [path]``/var/spacewalk``
| VirtIO Disk 3       | 50 GB for [path]``/var/lib/pgsql``
| VirtIO Disk 4       | 4 GB for swap
| Name:               | test-setup
| Network             | Bridge _br0_
|===

.{sle} Virtualization Guide

> [TIP]

Reviewer: I don't think this needs to be in an admonition. Plus, we don't use the [TIP] admonition.

| VirtIO Disk 4        | swap      | 4{nbsp}GB
|===

> . Click btn:[Begin Installation] and your new VM will boot from the JeOS image.

Reviewer: ```suggestion
. Click btn:[Begin Installation] to boot the new VM from the JeOS image.
```

| VirtIO Disk 4        | swap      | 4{nbsp}GB
|===

. Click btn:[Begin Installation] to boot the new VM from the JeOS image.

Proceed through the basic JeOS installation prompts until you reach the command line.

During the basic installation prompts you are asked to enter the root password.

> Select a strong password and in the next message box click btn:[Confirm root Password].

Reviewer: "Select" probably isn't the right verb here. Possibly "type"?

Author: I think we can shorten it.  Maybe, somewhere else we should talk about passwords in general.
